### PR TITLE
Fix editorial issues with equivalence properties.

### PR DESCRIPTION
**Pull request by msporny**
Partial editorial cleanup to appendices tracked as issue #728.

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/did-core/pull/749.html" title="Last updated on May 30, 2021, 3:00 PM UTC (e662df7)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/did-core/749/435a304...e662df7.html" title="Last updated on May 30, 2021, 3:00 PM UTC (e662df7)">Diff</a>

### CODE REVIEW

**Review by csuwildcat — CHANGES_REQUESTED** [review]
I don't understand why there are any special security issues with equivalent/canonical, because they are Method-specified. There's no difference in a Method resolving the initial ID or the equivalent/canonical variants; trust in the Method is the same wither way. The unique security issues are with alsoKnownAs, so I don't agree with them being lumped under this wording.

**Review by brentzundel — APPROVED** [review]

**Comment by csuwildcat** [comment]
@msporny the most significant nit is with the first part of the text. Shouldn't it say something like: "Given the `equivalentId` and `canonicalId` properties are determined by and within Methods themselves, they are subject to the same security guarantees a Method provides for the resolved ID itself, namely: if you trust a Method to properly resolve IDs in general, you trust the resolution of the IDs in these properties. The `alsoKnownAs` property is not Method-secured or guaranteed by Methods to be an accurate statement of equivalence by the controller of the resolved ID, thus they cannot be immediately relied upon without some Method-external validation procedure."

**Comment by msporny** [comment]
> @msporny the most significant nit is with the first part of the text. Shouldn't it say something like: "Given the `equivalentId` and `canonicalId` properties are determined by and within Methods themselves, they are subject to the same security guarantees a Method provides for the resolved ID itself, namely: if you trust a Method to properly resolve IDs in general, you trust the resolution of the IDs in these properties. The `alsoKnownAs` property is not Method-secured or guaranteed by Methods to be an accurate statement of equivalence by the controller of the resolved ID, thus they cannot be immediately relied upon without some Method-external validation procedure."

Yes, I'd be fine w/ that language. Please simplify it to a high school reading level -- that's what these editorial passes are about. It takes quite a bit of effort to parse your statement above... it's has a lot of embellishment in it... please try to strip it down to it's most basic components/statements.

**Review by kdenhartog — APPROVED** [review]

**Comment by csuwildcat** [comment]
How about this: "Given the equivalentId and canonicalId properties are determined by Methods themselves, the same security and accuracy guarantees that apply to the resolved ID present in the `id` field of a DID Document also apply to these properties. The `alsoKnownAs` property is not guaranteed to be an accurate statement of equivalence, and should not be relied upon without performing validation steps beyond the resolution of the DID Document."

**Comment by msporny** [comment]
> How about this: "Given the equivalentId and canonicalId properties are determined by Methods themselves, the same security and accuracy guarantees that apply to the resolved ID present in the `id` field of a DID Document also apply to these properties. The `alsoKnownAs` property is not guaranteed to be an accurate statement of equivalence, and should not be relied upon without performing validation steps beyond the resolution of the DID Document."

Perfect. Do you mind doing a change suggestion on this PR to that effect?

File: index.html [hunk 1]
```diff
@@ -4595,32 +4595,32 @@ <h2>Encrypted Data in DID Documents</h2>
     <section>
       <h2>Equivalence Properties</h2>
       <p>
-The three equivalence properties—<code><a>alsoKnownAs</a></code>, <code><a>equivalentId</a></code>, and
-<code><a>canonicalId</a></code>—are subject to special security considerations
-related to attacks against <a>DIDs</a> that are asserted to be equivalent.
+The three equivalence properties &mdash; <code><a>alsoKnownAs</a></code>,
+<code><a>equivalentId</a></code>, and <code><a>canonicalId</a></code> &mdash; are
+subject to special security considerations related to attacks against
+<a>DIDs</a> that are asserted to be equivalent.
```
Comment (csuwildcat) [inline index.html:4601]:
```suggestion
Given the <code><a>equivalentId</a></code> and <code><a>canonicalId</a></code> properties are determined by Methods themselves, the same security and accuracy guarantees that apply to the resolved ID present in the <code>id</code> field of a DID Document also apply to these properties. The <code><a>alsoKnownAs property is not guaranteed to be an accurate statement of equivalence, and should not be relied upon without performing validation steps beyond the resolution of the DID Document.
```

Comment (msporny) [reply index.html]: Accepted your change with one minor edit.

**Comment by msporny** [comment]
Editorial, multiple reviews, changes requested and made, no objections, merging.

**Pull request merged**